### ISSUE TITLE
missing texture

### ISSUE THREAD

**Issue by gionko** [treyyoder/quakejs-docker#17]
some map (like all "pro-..." map) missing texture in game.

**Comment by kwadroke** [comment]
That's due to the .pk3 files (demo + pak1-8) not having all of the textures from the retail version of Quake III Arena. Not really an issue with the docker container. The same issue exists with Quake3/ioQuake3 using the demo files.
The XCSV Texture pack fixes some, but not all of these. 
I've been working on a missing texture pack for a while now. Not quite complete yet.

**Comment by mrtheyann** [comment]
any updates on these?

**Comment by kwadroke** [comment]
A possible temporary solution would be to download the following file and put in `baseq3`
http://download.tuxfamily.org/openarena/files/pk3/q3a2oa/z-q3a2oa-textures-v4-080.pk3
Rename it to aaaa-q3a2oa-textures-v4-080.pk3 so it gets loaded first and the textures can be overridden.
The textures aren't pretty, but, you shouldn't get the missing texture lines.

I'm still working on my texture pack, and will eventually have a git repo. I am redoing them as Vector graphics so that the textures can be released in higher resolution versions. The goal is to be able to play Q3maps from places like LVLworld without any of the Quake III Arena pk3s. This will take a while.

**Comment by kwadroke** [comment]
Technically the real fix would to use the retail version of Quake III Arena pak0.pk3 on the client.

**Comment by gionko** [comment]
> A possible temporary solution would be to download the following file and put in `baseq3` http://download.tuxfamily.org/openarena/files/pk3/q3a2oa/z-q3a2oa-textures-v4-080.pk3 Rename it to aaaa-q3a2oa-textures-v4-080.pk3 so it gets loaded first and the textures can be overridden. The textures aren't pretty, but, you shouldn't get the missing texture lines. This is only needed client side unless you want users to download it from your server.
> 
> I'm still working on my texture pack, and will eventually have a git repo. I am redoing them as Vector graphics so that the textures can be released in higher resolution versions. The goal is to be able to play Q3maps from places like LVLworld without any of the Quake III Arena pk3s. This will take a while.

this solution not work.

**Comment by gionko** [comment]
https://github.com/gionko/content.quakejs.com <--- the solution for posterity

**Comment by kwadroke** [comment]
> This solution does not work because the repack of textures (assets) via repack.js is not performed.

That's true, you will need to use repack.js to "convert" it. (I'm getting the different Git repos for Quake 3 mixed up. and been using a standard Quake3 client server lately) 

> https://github.com/gionko/content.quakejs.com <--- the solution for posterity

Can you explain how this fixes the issue? At a quick look, It appears that it's just the contents of `contents.quakejs.com`.

**Comment by danpowell88** [comment]
Any updates on this?
